<?xml version="1.0" encoding="utf-8"?>
<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
  <file datatype="xml" original="HttpParseException.xml" source-language="en-US" target-language="ja-JP">
    <header>
      <tool tool-id="mdxliff" tool-name="mdxliff" tool-version="1.0-15c36f0" tool-company="Microsoft" />
      <xliffext:skl_file_name xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">02cd5861-7ce2-4a82-b358-31f8435a0ac5ba8323b1bf107e6f8daae03912597f3e0ed37ca4.skl</xliffext:skl_file_name>
      <xliffext:version xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">1.2</xliffext:version>
      <xliffext:ms.openlocfilehash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">ba8323b1bf107e6f8daae03912597f3e0ed37ca4</xliffext:ms.openlocfilehash>
      <xliffext:ms.sourcegitcommit xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">d31dc2ede16f6f7bc64e90d9f897ff54c4e3869b</xliffext:ms.sourcegitcommit>
      <xliffext:ms.lasthandoff xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">04/03/2018</xliffext:ms.lasthandoff>
      <xliffext:moniker_ids xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">netframework-4.5.1,netframework-4.5.2,netframework-4.5,netframework-4.6.1,netframework-4.6.2,netframework-4.6,netframework-4.7.1,netframework-4.7</xliffext:moniker_ids>
    </header>
    <body>
      <group id="content" extype="content">
        <trans-unit id="101" translate="yes" xml:space="preserve" uid="T:System.Web.HttpParseException">
          <source>The exception that is thrown when a parse error occurs.</source>
          <target state="translated">解析エラーが発生したときにスローされる例外。</target>       </trans-unit>
        <trans-unit id="102" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Web.HttpParseException">
          <source>The <ph id="ph1">&lt;xref:System.Web.HttpParseException&gt;</ph> class is an HTTP-specific exception class that enables ASP.NET to output parser exception information.</source>
          <target state="translated"><ph id="ph1">&lt;xref:System.Web.HttpParseException&gt;</ph>クラスは、HTTP 固有の例外クラスをパーサー例外情報を出力する ASP.NET を有効にします。</target>       </trans-unit>
        <trans-unit id="103" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Web.HttpParseException">
          <source>For more information on throwing and handling exceptions, see <bpt id="p1">[</bpt>Exceptions<ept id="p1">](http://msdn.microsoft.com/library/f99a1d29-a2a8-47af-9707-9909f9010735)</ept>.</source>
          <target state="translated">スローして、例外の処理の詳細については、次を参照してください。<bpt id="p1">[</bpt>例外<ept id="p1">](http://msdn.microsoft.com/library/f99a1d29-a2a8-47af-9707-9909f9010735)</ept>です。</target>       </trans-unit>
        <trans-unit id="104" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Web.HttpParseException">
          <source>The following example demonstrates how to use the <ph id="ph1">&lt;xref:System.Web.HttpParseException&gt;</ph> to customize errors generated during page parsing.</source>
          <target state="translated">次の例で使用する方法、<ph id="ph1">&lt;xref:System.Web.HttpParseException&gt;</ph>をページの解析中に生成されたエラーをカスタマイズします。</target>       </trans-unit>
        <trans-unit id="105" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Web.HttpParseException">
          <source>In this example, a customized <ph id="ph1">&lt;xref:System.Web.UI.HtmlControls.HtmlSelect&gt;</ph> control is defined.</source>
          <target state="translated">この例で、カスタマイズされた<ph id="ph1">&lt;xref:System.Web.UI.HtmlControls.HtmlSelect&gt;</ph>コントロールが定義されています。</target>       </trans-unit>
        <trans-unit id="106" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Web.HttpParseException">
          <source>If child elements of the custom control are not of a specified type then an <ph id="ph1">&lt;xref:System.Web.HttpParseException&gt;</ph> is thrown in the overridden <ph id="ph2">&lt;xref:System.Web.UI.HtmlControls.HtmlSelectBuilder.GetChildControlType%2A&gt;</ph> method of a custom <ph id="ph3">&lt;xref:System.Web.UI.HtmlControls.HtmlSelectBuilder&gt;</ph>.</source>
          <target state="translated">カスタム コントロールの子要素は、指定した型のないかどうか、<ph id="ph1">&lt;xref:System.Web.HttpParseException&gt;</ph>がスローされます、オーバーライドされたで<ph id="ph2">&lt;xref:System.Web.UI.HtmlControls.HtmlSelectBuilder.GetChildControlType%2A&gt;</ph>カスタム メソッド<ph id="ph3">&lt;xref:System.Web.UI.HtmlControls.HtmlSelectBuilder&gt;</ph>です。</target>       </trans-unit>
        <trans-unit id="107" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Web.HttpParseException">
          <source>To generate a parse exception, change the child element literal <ph id="ph1">`MyCustomOption`</ph> to any other string.</source>
          <target state="translated">解析の例外を生成するリテラルの子要素を変更する<ph id="ph1">`MyCustomOption`</ph>他の任意の文字列にします。</target>       </trans-unit>
        <trans-unit id="108" translate="yes" xml:space="preserve" uid="T:System.Web.HttpParseException">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Web.HttpParseException" /&gt;</ph> class.</source>
          <target state="translated"><ph id="ph1">&lt;see cref="T:System.Web.HttpParseException" /&gt;</ph> クラスの新しいインスタンスを初期化します。</target>       </trans-unit>
        <trans-unit id="109" translate="yes" xml:space="preserve" uid="M:System.Web.HttpParseException.#ctor">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Web.HttpParseException" /&gt;</ph> class.</source>
          <target state="translated"><ph id="ph1">&lt;see cref="T:System.Web.HttpParseException" /&gt;</ph> クラスの新しいインスタンスを初期化します。</target>       </trans-unit>
        <trans-unit id="110" translate="yes" xml:space="preserve" uid="M:System.Web.HttpParseException.#ctor(System.String)">
          <source>The exception message to specify when the error occurs.</source>
          <target state="translated">エラーの発生時に指定する例外メッセージ。</target>       </trans-unit>
        <trans-unit id="111" translate="yes" xml:space="preserve" uid="M:System.Web.HttpParseException.#ctor(System.String)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Web.HttpParseException" /&gt;</ph> class with a specified error message.</source>
          <target state="translated">指定したエラー メッセージを使用して、<ph id="ph1">&lt;see cref="T:System.Web.HttpParseException" /&gt;</ph> クラスの新しいインスタンスを初期化します。</target>       </trans-unit>
        <trans-unit id="112" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpParseException.#ctor(System.String)">
          <source>The following example demonstrates the use of the <ph id="ph1">&lt;xref:System.Web.HttpException.%23ctor%2A&gt;</ph> constructor.</source>
          <target state="translated">次の例での使用、<ph id="ph1">&lt;xref:System.Web.HttpException.%23ctor%2A&gt;</ph>コンス トラクターです。</target>       </trans-unit>
        <trans-unit id="113" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpParseException.#ctor(System.String)">
          <source>For a complete, working code example, see the <ph id="ph1">&lt;xref:System.Web.HttpParseException&gt;</ph> overview topic.</source>
          <target state="translated">完了したら、作業中のコード例は、次を参照してください。、<ph id="ph1">&lt;xref:System.Web.HttpParseException&gt;</ph>概要トピックです。</target>       </trans-unit>
        <trans-unit id="114" translate="yes" xml:space="preserve" uid="M:System.Web.HttpParseException.#ctor(System.String,System.Exception)">
          <source>The exception message to specify when the error occurs.</source>
          <target state="translated">エラーの発生時に指定する例外メッセージ。</target>       </trans-unit>
        <trans-unit id="115" translate="yes" xml:space="preserve" uid="M:System.Web.HttpParseException.#ctor(System.String,System.Exception)">
          <source>The exception that is the cause of the current exception.</source>
          <target state="translated">現在の例外の原因となった例外。</target>       </trans-unit>
        <trans-unit id="116" translate="yes" xml:space="preserve" uid="M:System.Web.HttpParseException.#ctor(System.String,System.Exception)">
          <source>If the <bpt id="p1">&lt;c&gt;</bpt>innerException<ept id="p1">&lt;/c&gt;</ept> parameter is not <ph id="ph1">&lt;see langword="null" /&gt;</ph>, the current exception is raised in a <ph id="ph2">&lt;see langword="catch" /&gt;</ph> block that handles the inner exception.</source>
          <target state="translated"><bpt id="p1">&lt;c&gt;</bpt>innerException<ept id="p1">&lt;/c&gt;</ept> パラメーターが <ph id="ph1">&lt;see langword="null" /&gt;</ph> でない場合は、内部例外を処理する <ph id="ph2">&lt;see langword="catch" /&gt;</ph> ブロックで現在の例外が発生します。</target>       </trans-unit>
        <trans-unit id="117" translate="yes" xml:space="preserve" uid="M:System.Web.HttpParseException.#ctor(System.String,System.Exception)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Web.HttpParseException" /&gt;</ph> class with a specified error message and a reference to the inner.</source>
          <target state="translated">指定したエラー メッセージと内部例外に対する参照を使用して、<ph id="ph1">&lt;see cref="T:System.Web.HttpParseException" /&gt;</ph> クラスの新しいインスタンスを初期化します。</target>       </trans-unit>
        <trans-unit id="118" translate="yes" xml:space="preserve" uid="M:System.Web.HttpParseException.#ctor(System.String,System.Exception,System.String,System.String,System.Int32)">
          <source>The exception message to specify when the error occurs.</source>
          <target state="translated">エラーの発生時に指定する例外メッセージ。</target>       </trans-unit>
        <trans-unit id="119" translate="yes" xml:space="preserve" uid="M:System.Web.HttpParseException.#ctor(System.String,System.Exception,System.String,System.String,System.Int32)">
          <source>The exception that is the cause of the current exception.</source>
          <target state="translated">現在の例外の原因となった例外。</target>       </trans-unit>
        <trans-unit id="120" translate="yes" xml:space="preserve" uid="M:System.Web.HttpParseException.#ctor(System.String,System.Exception,System.String,System.String,System.Int32)">
          <source>If <bpt id="p1">&lt;c&gt;</bpt>innerException<ept id="p1">&lt;/c&gt;</ept> is not <ph id="ph1">&lt;see langword="null" /&gt;</ph>, the current exception is raised in a <ph id="ph2">&lt;see langword="catch" /&gt;</ph> block that handles the inner exception.</source>
          <target state="translated"><bpt id="p1">&lt;c&gt;</bpt>innerException<ept id="p1">&lt;/c&gt;</ept> が <ph id="ph1">&lt;see langword="null" /&gt;</ph> でない場合は、内部例外を処理する <ph id="ph2">&lt;see langword="catch" /&gt;</ph> ブロックで現在の例外が発生します。</target>       </trans-unit>
        <trans-unit id="121" translate="yes" xml:space="preserve" uid="M:System.Web.HttpParseException.#ctor(System.String,System.Exception,System.String,System.String,System.Int32)">
          <source>The virtual path for the exception.</source>
          <target state="translated">例外の仮想パス。</target>       </trans-unit>
        <trans-unit id="122" translate="yes" xml:space="preserve" uid="M:System.Web.HttpParseException.#ctor(System.String,System.Exception,System.String,System.String,System.Int32)">
          <source>The source code being compiled when the exception occurs.</source>
          <target state="translated">エラーの発生時にコンパイルされていたソース コード。</target>       </trans-unit>
        <trans-unit id="123" translate="yes" xml:space="preserve" uid="M:System.Web.HttpParseException.#ctor(System.String,System.Exception,System.String,System.String,System.Int32)">
          <source>The line number on which the exception occurred.</source>
          <target state="translated">例外が発生した行番号。</target>       </trans-unit>
        <trans-unit id="124" translate="yes" xml:space="preserve" uid="M:System.Web.HttpParseException.#ctor(System.String,System.Exception,System.String,System.String,System.Int32)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Web.HttpParseException" /&gt;</ph> class with specific information about the source code being compiled and the line number on which the exception occurred.</source>
          <target state="translated">コンパイルされていたソース コードに関する情報と例外が発生した行番号を使用して、<ph id="ph1">&lt;see cref="T:System.Web.HttpParseException" /&gt;</ph> クラスの新しいインスタンスを初期化します。</target>       </trans-unit>
        <trans-unit id="125" translate="yes" xml:space="preserve" uid="P:System.Web.HttpParseException.FileName">
          <source>Gets the name of the file being parsed when the error occurs.</source>
          <target state="translated">エラーの発生時に解析されていたファイルの名前を取得します。</target>       </trans-unit>
        <trans-unit id="126" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Web.HttpParseException.FileName">
          <source>The physical path to the source file that is being parsed when the error occurs; otherwise, <ph id="ph1">&lt;see langword="null" /&gt;</ph>, if the physical path is <ph id="ph2">&lt;see langword="null" /&gt;</ph>.</source>
          <target state="translated">エラー発生時に解析中だったソース ファイルへの物理パス。物理パスが <ph id="ph1">&lt;see langword="null" /&gt;</ph> の場合は <ph id="ph2">&lt;see langword="null" /&gt;</ph>。</target>       </trans-unit>
        <trans-unit id="127" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Web.HttpParseException.FileName">
          <source>The <ph id="ph1">&lt;xref:System.Web.HttpParseException.FileName%2A&gt;</ph> property returns the physical path to the source file that has the error unless the physical path is <ph id="ph2">`null`</ph>, in which case it returns <ph id="ph3">`null`</ph>.</source>
          <target state="translated"><ph id="ph1">&lt;xref:System.Web.HttpParseException.FileName%2A&gt;</ph>プロパティは、物理パスがない限り、エラーのあるソース ファイルを物理パスを返します<ph id="ph2">`null`</ph>、後者を返します<ph id="ph3">`null`</ph>です。</target>       </trans-unit>
        <trans-unit id="128" translate="yes" xml:space="preserve" uid="M:System.Web.HttpParseException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.SerializationInfo" /&gt;</ph> that holds the serialized object data about the exception being thrown.</source>
          <target state="translated">スローされている例外に関するシリアル化済みオブジェクト データを保持している <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.SerializationInfo" /&gt;</ph>。</target>       </trans-unit>
        <trans-unit id="129" translate="yes" xml:space="preserve" uid="M:System.Web.HttpParseException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.StreamingContext" /&gt;</ph> that contains contextual information about the source or destination.</source>
          <target state="translated">転送元または転送先についてのコンテキスト情報を含む <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.StreamingContext" /&gt;</ph> です。</target>       </trans-unit>
        <trans-unit id="130" translate="yes" xml:space="preserve" uid="M:System.Web.HttpParseException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>When overridden in a derived class, sets the <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.SerializationInfo" /&gt;</ph> object with information about the exception.</source>
          <target state="translated">派生クラスでオーバーライドされた場合は、その例外に関する情報を使用して <ph id="ph1">&lt;see cref="T:System.Runtime.Serialization.SerializationInfo" /&gt;</ph> オブジェクトを設定します。</target>       </trans-unit>
        <trans-unit id="131" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpParseException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The <ph id="ph1">&lt;xref:System.Web.HttpParseException.GetObjectData%2A&gt;</ph> method sets a <ph id="ph2">&lt;xref:System.Runtime.Serialization.SerializationInfo&gt;</ph> object with all the exception object data targeted for serialization.</source>
          <target state="translated"><ph id="ph1">&lt;xref:System.Web.HttpParseException.GetObjectData%2A&gt;</ph>メソッドのセット、<ph id="ph2">&lt;xref:System.Runtime.Serialization.SerializationInfo&gt;</ph>シリアル化の対象となるすべての例外オブジェクト データを持つオブジェクト。</target>       </trans-unit>
        <trans-unit id="132" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpParseException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>During deserialization, the exception is reconstituted from the SerializationInfo transmitted over the stream.</source>
          <target state="translated">逆シリアル化中には、ストリームで送信された SerializationInfo から例外が再構成されます。</target>       </trans-unit>
        <trans-unit id="133" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpParseException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The following code example shows how to define a derived serializable <ph id="ph1">&lt;xref:System.Exception&gt;</ph> class that implements the <ph id="ph2">&lt;xref:System.Web.HttpParseException.GetObjectData%2A&gt;</ph> method, which makes minor changes to two properties, and then calls the base class to perform the serialization.</source>
          <target state="translated">次のコード例は、派生を定義する方法を示しています。 シリアル化可能な<ph id="ph1">&lt;xref:System.Exception&gt;</ph>を実装するクラス、<ph id="ph2">&lt;xref:System.Web.HttpParseException.GetObjectData%2A&gt;</ph>メソッドでは、2 つのプロパティに軽微な変更を行い、シリアル化を実行する基本クラスを呼び出します。</target>       </trans-unit>
        <trans-unit id="134" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpParseException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The example forces a divide-by-0 error, and then creates an instance of the derived <ph id="ph1">&lt;xref:System.Exception&gt;</ph>.</source>
          <target state="translated">例では、強制的に 0 による除算のエラー、および派生クラスのインスタンスを作成し<ph id="ph1">&lt;xref:System.Exception&gt;</ph>です。</target>       </trans-unit>
        <trans-unit id="135" translate="yes" xml:space="preserve" extradata="MT" uid="M:System.Web.HttpParseException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The code serializes the instance to a file, deserializes the file into a new <ph id="ph1">&lt;xref:System.Exception&gt;</ph>, which it throws, and then catches and displays the exception's data.</source>
          <target state="translated">コードをファイルにインスタンスをシリアル化する、新しいファイルに逆シリアル化<ph id="ph1">&lt;xref:System.Exception&gt;</ph>がスローおよびキャッチし、例外のデータが表示されます。</target>       </trans-unit>
        <trans-unit id="136" translate="yes" xml:space="preserve" uid="M:System.Web.HttpParseException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)">
          <source>The <ph id="ph1">&lt;paramref name="info" /&gt;</ph> parameter is <ph id="ph2">&lt;see langword="null" /&gt;</ph>.</source>
          <target state="translated"><ph id="ph1">&lt;paramref name="info" /&gt;</ph> パラメーターが <ph id="ph2">&lt;see langword="null" /&gt;</ph> です。</target>       </trans-unit>
        <trans-unit id="137" translate="yes" xml:space="preserve" uid="P:System.Web.HttpParseException.Line">
          <source>Gets the number of the line being parsed when the error occurs.</source>
          <target state="translated">エラーの発生時に解析されていた行の番号を取得します。</target>       </trans-unit>
        <trans-unit id="138" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Web.HttpParseException.Line">
          <source>The number of the line being parsed when the error occurs.</source>
          <target state="translated">エラーの発生時に解析されていた行の番号。</target>       </trans-unit>
        <trans-unit id="139" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Web.HttpParseException.Line">
          <source>This value is 1-based, not 0-based.</source>
          <target state="translated">値は、0 からではなく、1 から始まります。</target>       </trans-unit>
        <trans-unit id="140" translate="yes" xml:space="preserve" uid="P:System.Web.HttpParseException.ParserErrors">
          <source>Gets the parser errors for the current exception.</source>
          <target state="translated">現在の例外の解析エラーを取得します。</target>       </trans-unit>
        <trans-unit id="141" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Web.HttpParseException.ParserErrors">
          <source>A collection of the parser errors for the current exception.</source>
          <target state="translated">現在の例外の解析エラーのコレクション。</target>       </trans-unit>
        <trans-unit id="142" translate="yes" xml:space="preserve" uid="P:System.Web.HttpParseException.VirtualPath">
          <source>Gets the virtual path to source file that generated the error.</source>
          <target state="translated">エラーを生成したソース ファイルへの仮想パスを取得します。</target>       </trans-unit>
        <trans-unit id="143" translate="yes" xml:space="preserve" extradata="MT" uid="P:System.Web.HttpParseException.VirtualPath">
          <source>The virtual path to the source file that generated the error.</source>
          <target state="translated">エラーを生成したソース ファイルへの仮想パス。</target>       </trans-unit>
      </group>
    </body>
  </file>
</xliff>